.travis.yml

```diff
@@ -34,8 +34,7 @@ before_deploy:
   - tar xzvf $HOME/cf.tgz -C $HOME
   - mkdir -p ${HOME}/Godeps/_workspace
   - export GOPATH=${HOME}/Godeps/_workspace
```

Comment:
and remove this line. since you're not compiling from source, no need to make a `Go` workspace